README.md

```diff
@@ -30,9 +33,9 @@ apt install build-essential autoconf automake libtool m4 libopus-dev youtube-dl
 ### Windows
 
 If you are using the MSVC toolchain, a prebuilt DLL for Opus is already provided for you.  
-You will only need to download [FFmpeg](https://ffmpeg.org/download.html), and install youtube-dl which can be done through Python's package manager, pip.
+You will only need to download [FFmpeg](https://ffmpeg.org/download.html), and install [yt-dlp](https://github.com/yt-dlp/yt-dlp) which can be done through Python's package manager, pip.
 ```shell
-pip install youtube_dl
+pip install -U yt-dlp
 ```
```

Comment:
I would even go ahead and delete this, as the ffmpeg and yt-dlp readmes already tell you how to install

Reply:
I would be in favour of simplifying the README and delegate install instructions to the respective repos. Maybe we do it at a later time?